modules/percolator/src/main/java/org/elasticsearch/percolator/PercolatorFieldMapper.java

```diff
@@ -270,7 +270,9 @@ public Mapper parse(ParseContext context) throws IOException {
         }
 
         XContentParser parser = context.parser();
-        QueryBuilder queryBuilder = parseQueryBuilder(queryShardContext.newParseContext(parser), parser.getTokenLocation());
+        QueryBuilder queryBuilder = parseQueryBuilder(
+                queryShardContext.newParseContext(parser), parser.getTokenLocation()
```

Comment:
The `PercolatorBackwardsCompatibilityTests` already exists for verifying that pre 5 indices remain to work. My plan is to adjust this test, so that we also check we do the right things for scripts in percolator query with implicit default lang.